test/e2e/specs/editor/editor__post-basic-flow.ts

```diff
@@ -86,7 +86,11 @@ describe( DataHelper.createSuiteTitle( 'Editor: Basic Post Flow' ), function ()
 
 	describe( 'Jetpack features', function () {
 		it( 'Open Jetpack settings', async function () {
-			await editorPage.openSettings( 'Jetpack' );
+			await editorPage.openEditorOptionsMenu();
```

Comment:
`openSettings` doesn't work for items inside the toolbar "options" menu. It doesn't look like changing it to support dropdown items would be a good idea, mostly because it seems to couple with the button aria "expanding" logic, which AFAIK wouldn't be the case with the dropdown item (might be wrong).

An idea might be to refactor the "openSettings" method to just always use the "options" dropdown, to avoid the whole problem with pinned buttons. The plugin buttons are always rendered in the options dropdown, regardless of the viewport size, it seems (see below).

The TL;DR is: this logic works but we might to give more thought on how to better abstract this. It's also important to note that the Desktop viewport is also opening the Jetpack settings by going to the "Options" menu first. I left it as is because it works, but le time know if you want to keep clicking the jetpack pinned button instead.